### PR TITLE
Add Carbonfuture

### PR DESCRIPTION
**Pull request by 1ea**
- [x] I read [CONTRIBUTING.md](https://github.com/pogopaule/awesome-sustainability-jobs/blob/main/CONTRIBUTING.md)  
- [x] I didn't change README.md (but instead src/data.yaml or src/template.md)

For transparency note: I work for carbonfuture. 
We built a platform for high quality carbon removal and bring together sink producers and carbon credit buyers.

### CODE REVIEW

File: src/data.yaml [hunk 1]
```diff
@@ -2575,6 +2575,19 @@ jobs:
...
+    jobs: https://www.carbonfuture.earth/jobs
+    description: Carbon Removal Platform
+    remote: true
+    speculative: true
+    field: Carbon Analysis, Capture and Offsetting
+    geo:
+      - country: Germany
+        lat: 48.0145161
+        long: 7.8449299
+        review:
```
Comment (pogopaule) [inline src/data.yaml]: The indentation is off for review and rating

Comment (1ea) [reply src/data.yaml]: oops, sorry! Fixed it.

Comment (pogopaule) [reply src/data.yaml]: Danke!

**Comment by pogopaule** [comment]
Besten Dank für den PR!

**Comment by pogopaule** [comment]
Besten Dank für den PR!

**Pull request merged**